src/SampleViewExtension/SampleViewExtension.csproj

```diff
@@ -161,10 +161,14 @@
     <None Include="packages.config" />
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <PropertyGroup>
-    <PostBuildEvent>xcopy "$(ProjectDir)bin\Debug\SampleViewExtension.dll" "$(ProjectDir)..\..\dynamo_viewExtension\Sample View Extension\bin" /Y /E
-xcopy "$(ProjectDir)bin\Debug\SampleViewExtension_ViewExtensionDefinition.xml" "$(ProjectDir)..\..\dynamo_viewExtension\Sample View Extension\extra" /Y /E</PostBuildEvent>
-  </PropertyGroup>
+  <Target Name="AfterBuild">
+    <ItemGroup>
+      <PackageDll Include="$(ProjectDir)bin\Debug\SampleViewExtension.dll" />
+      <PackageXml Include="$(ProjectDir)bin\Debug\SampleViewExtension_ViewExtensionDefinition.xml" />
```

Comment:
Don't use `Debug` but target `Release`, after reading the console log. I think the problem is that the whole solution is built using release so if you search for the dll under debug folder, it will not be there.

Reply:
This maybe broken because at some point we changed default to build on release instead of debug? I can't tell the history but from the log, that seems why it is failing now

Reply:
Thanks @QilongTang!  I switched to using `$(ConfigurationName)` in the path so we handle debug and release the same in regards to the post-build copying in the package compilation step.